src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
I would go with dashboard,  to be fair, maybe rename the no param version
to dashboards
On 12 Mar 2015 7:59 PM, "Eddie Abou-Jaoude" <notifications@github.com>
wrote:

> In src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml
> <https://github.com/DashboardHub/PipelineDashboard/pull/75#discussion_r26338438>
> :
>
> > @@ -2,6 +2,10 @@ dashboardhub_app_homepage:
> >      path:     /
> >      defaults: { _controller: DashboardHubAppBundle:Default:index }
> >
> > +dashboardhub_app_dashboard:
> > +    path:     /d/{uid}
>
> dash?
>
> Which will become... dashboardhub.io/dash/{key}
> <http://dashboardhub.io/dash/%7Bkey%7D>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/75/files#r26338438>
> .
>